deployment/arm/deploy-region.sh

```diff
@@ -7,8 +7,10 @@ appName=$2
 env=$3
```

Comment:
add flag "set -e"

Reply:
I have added set -eu to all bash scripts.